.ci/common/test.sh

```diff
@@ -73,6 +73,11 @@ run_oldtests() {
   valgrind_check "${LOG_DIR}"
 }
 
+run_single_includes_tests() {
```

Comment:
this should be called from .ci/common/run_test.sh,
but removed temporarily based on the information from @ZyX-I as follows.

> AFAIR my work resulted in some weird CI failures (don’t remember the details) and thus these tests were disabled. 

If the policy in neovim forbid any inactive changes to be merged, I would like to try to activate the changes.